graph/src/env/mod.rs

```diff
@@ -17,6 +18,7 @@ use crate::{
 
 lazy_static! {
     pub static ref ENV_VARS: EnvVars = EnvVars::from_env().unwrap();
+    pub static ref TEST_WITH_NO_REORG: Mutex<bool> = Mutex::new(false);
```

Comment:
This is minor, but I think we could also compile this out with `#[cfg(not(debug_assertions))]`